README.md

```diff
@@ -197,7 +197,7 @@ index.search(
 
 ## 🤖 Compatibility with Meilisearch
 
-This package only guarantees the compatibility with the [version v0.30.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.30.0).
+This package only guarantees the compatibility with the [version v1.0.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v1.0.0).
```

Comment:
Should this just be `v1` instead of `v1.0.0`? At this point any breaking change that would stop the SDK from working would be `v2` of Meilisearch so this SDK should work with any `v1.x.x` version.

Reply:
Thanks, @sanders41. You're absolutely right, and I'm happy with the change to only have the major version from now on. I will change that